### ISSUE TITLE
Suggestion: Make in Modding Queue forum's posts button "Done"

### ISSUE THREAD

**Issue by 3mplify** [ppy/osu-web#2807]
https://i.imgur.com/6gwyC5W.png
1) It would be nice feedback 
2) It would safe pretty much time, if you did modding in different order

**Comment by peppy** [comment]
I'm not sure what you are saying here. Could you please explain what this "done" button would do, and who would see it?

**Comment by 3mplify** [comment]
I was modding some maps, but did it in random order. So, when I checked
modding queue again I forgot which mods I've done. And I thought it would
be nice for people who mod maps to have something to mark posts, where the
mod was done.
I see that as "status of mod" or something like that. For example, no to
shitpost author just click the button and mapper will notice that his mod
was accepted and it's in progress. When mod finished - author change status
to "done". If he don't like map he change status to "denied".
I find this stuff helpful for both sides.
To be honest, I want to see upgrade of modding queue. It looks just like
the forum. Maybe it would be better to change writting posts to forms with
info about beatmaps (like when you registred to sites lel), which can be
modified by author of queue. Also would be nice if author could restrict
ammount of this forms, because a lot of mappers take 2-4 maps at a time.
Maybe this form stuff sounds a bit complicated, but if you interested I'm
ready to write a huge post with pictures about how I see that, and why it's
cool stuff.
Also, would be nice to divide queues for "opened" and "closed". 1) This
prevent author shitposting like "queue is closed" or "queue is opened". 2)
I hate when I open queue from the list and found that it's closed.

In addition, I want to say that I hate that modding queue is just a forum.
You can make a function to rate a queue after getting a mod. (When author
gives a mod status done, the mapper who got mod takes a message to rate the
queue from 0 to 5 with 3 categories: speed, efficiency and *something about
how good that was explained, but I hadn't find how to name it*. For what we
need this? To fix modding queue list. Modding queue would be sorted by that
ratting and people would see who make better mods. Also this would increase
quality of modding, because people would work harder to get all 5*.
To sum up, I think modding queues should be much more than modding posts
with more efficient functions.
Hope you interested with it.
Best wishes,
Nikita "3mplify" Bevz.

чт, 29 мар. 2018 г., 9:45 Dean Herbert <notifications@github.com>:

**Comment by TPGPL** [comment]
should be closed as new site no longer supports modding v1, i think

**Comment by dorothy3242** [comment]
Unless there are plans to revamp the modding queues system (which there really should be, keeping it forum based seems weird next to modding v2 now) this suggestion has little to nothing to do with modding v1/v2 as the modding queues are separate..?

**Comment by TPGPL** [comment]
i think there's a planned migration from modding v1 to v2 for older beatmaps

**Comment by dorothy3242** [comment]
Modding queues is separate from beatmap modding.

**Comment by 3mplify** [comment]
Lmao, twas suggestion about upgrading modding queue, not beatmap's discussion
